servicetalk-examples/http/serialization/protobuf/src/main/java/io/servicetalk/examples/http/serialization/protobuf/SerializerUtils.java

```diff
@@ -40,28 +41,30 @@
  * utilities to concert it to HTTP-aware variants.
  */
 public final class SerializerUtils {
-
-    private static final CharSequence APPLICATION_PROTOBUF = newAsciiString("application/protobuf");
     private static final Consumer<HttpHeaders> CONTENT_TYPE_SETTER =
             headers -> headers.set(CONTENT_TYPE, APPLICATION_PROTOBUF);
     private static final Predicate<HttpHeaders> CONTENT_TYPE_VALIDATOR =
             headers -> hasContentType(headers, APPLICATION_PROTOBUF, null);
+    private static final Consumer<HttpHeaders> VAR_INT_CONTENT_TYPE_SETTER =
+            headers -> headers.set(CONTENT_TYPE, APPLICATION_PROTOBUF_VAR_INT);
+    private static final Predicate<HttpHeaders> VAR_INT_CONTENT_TYPE_VALIDATOR =
+            headers -> hasContentType(headers, APPLICATION_PROTOBUF_VAR_INT, null);
 
     public static final HttpSerializerDeserializer<RequestMessage> REQ_SERIALIZER =
             serializer(PROTOBUF.serializerDeserializer(RequestMessage.parser()),
                     CONTENT_TYPE_SETTER, CONTENT_TYPE_VALIDATOR);
 
     public static final HttpStreamingSerializerDeserializer<RequestMessage> REQ_STREAMING_SERIALIZER =
             streamingSerializer(PROTOBUF.streamingSerializerDeserializer(RequestMessage.parser()),
-                    CONTENT_TYPE_SETTER, CONTENT_TYPE_VALIDATOR);
+                    VAR_INT_CONTENT_TYPE_SETTER, VAR_INT_CONTENT_TYPE_VALIDATOR);
```

Comment:
@idelpivnitskiy - note this change. Streaming serialization with VarInt encoding is not the same as encoding/decoding single objects. Having a unique content type makes the exception more clear if you mix the examples (streaming client, scalar server).